### ISSUE TITLE
Tamano de area scaneo IOS

### ISSUE THREAD

**Issue by cabustos** [AmolGangadhare/flutter_barcode_scanner#195]
Un saludo cordial en , he usado el plugin y ha sido fabuloso en android escanea sin problema pero IOS ,he probado con un iphone 6, y no reconoce , pareciera cuando los codigos son muy anchos y hay que alejar la camara para enfocarlos no pudiera reconocerlos, se podria ampliar el ancho del area de scaneo al poner horizontal el celular

o si hay otra alternativa agradezco, actualmente lo uso en flutter pero IOS no me reconoce.

saludos y gracias infinitas por tu apoyo

**Comment by AmolGangadhare** [comment]
Thanks for using the plugin.

What I understood with google translate is the scanner is not working for iPhone 6, please acknowledge did I got it right.

Allow me some time to analyze this before making any comment at this moment.

Thanks

**Comment by cabustos** [comment]
Yes, if this is the case on iPhone 6 it does not recognize the pdf417 code, iOS 14

**Comment by AmolGangadhare** [comment]
Thanks for confirming, please allow me a little time to analyze this.

Gracias por confirmar, permítame un poco de tiempo para analizar esto.

**Comment by cabustos** [comment]
An additional piece of information, the code I am trying to read is wide, and to fit it into the recognition area I have to move the cell phone away and it does not recognize it, xq if I look for a smaller pdf417 code, less wide if it recognizes it
